src/result.rs

```diff
@@ -25,6 +25,7 @@ pub const Z_ENETWORK: z_result_t = -4;
 pub const Z_ENULL: z_result_t = -5;
 pub const Z_EUNAVAILABLE: z_result_t = -6;
 pub const Z_EDESERIALIZE: z_result_t = -7;
+pub const Z_SESSION_CLOSED: z_result_t = -8;
```

Comment:
Z_ESESSION_CLOSED maybe, to make in the same style as all other errors ?

Reply:
Done